vue3-apollo-quasar/src/components/ProfilePageLayout/ProfilePageLayout.vue

```diff
@@ -16,19 +16,69 @@
           <UserProfileCard :username="username" />
         </div>
         <!-- Right side -->
-        <div class="tab-contents col" v-if="!loadingLanguage">
+        <div v-if="!!isQuerying" class="tab-contents col">
```

Comment:
The name `isQuerying` implies that this is already a `boolean`. My suggestion would be to make it a `boolean` in the script rather than to cast it to one in the template.